### PR TITLE
Test fix pull request

### PR DESCRIPTION
**Pull request by EKarpulevich**
Test system are working from eclipse now.
Also you can run all existing tests from test/Main.java

### CODE REVIEW

File: src/test/java/com/google/security/zynamics/binnavi/testdata/database_schema.sql [hunk 1]
```diff
@@ -3450,41 +3450,57 @@ ALTER FUNCTION public.load_expression_type_instances(moduleid integer) OWNER TO
...
 --
--- Name: load_function_information(integer); Type: FUNCTION; Schema: public; Owner: postgres
+-- load_function_information(IN moduleid integer, IN address bigint)
 --
 
-CREATE FUNCTION load_function_information(moduleid integer) RETURNS TABLE(view_id integer, address bigint, name text, original_name text, description text, bbcount bigint, edgecount bigint, incount bigint, outcount bigint, global_comment integer, type function_type, parent_module_name text, parent_module_id integer, parent_module_function integer, stack_frame integer)
-    LANGUAGE sql
-    AS $_$
+CREATE OR REPLACE FUNCTION load_function_information(IN moduleid integer, IN address bigint)
+  RETURNS TABLE(
```
Comment (0xFFFFFFFE) [inline src/test/java/com/google/security/zynamics/binnavi/testdata/database_schema.sql]: Can you go into a little bit of detail why this is necessary beneficial?

File: src/test/java/com/google/security/zynamics/binnavi/testdata/database_schema.sql [hunk 2]
```diff
@@ -3497,80 +3513,81 @@ bc.bbcount, ec.edgecount, ic.incount, oc.outcount,
  JOIN function_edge_count AS ec ON ec.function = ft.address
  JOIN function_in_count AS ic ON ic.function = ft.address
  JOIN function_out_count AS oc ON oc.function = ft.address
- WHERE ft.module_id = $1
- ORDER BY ft.address
-
-$_$;
-
-
-ALTER FUNCTION public.load_function_information(moduleid integer) OWNER TO postgres;
```
Comment (0xFFFFFFFE) [inline src/test/java/com/google/security/zynamics/binnavi/testdata/database_schema.sql:3506]: Is there a reason to delete the ALTER to OWNER statements?

**Comment by EKarpulevich** [comment]
@0xFFFFFFFE 
About this:

> > -CREATE FUNCTION load_function_information
> > +CREATE OR REPLACE FUNCTION load_function_information
> 
> Can you go into a little bit of detail why this is necessary beneficial?

There was errors in several tests because "bn_functions" table in tests had no "prototype" field and "load_function_information" function had no return field "prototype".
So I added one empty field to "bn_functions" table and take"load_function_information" function from  src/main/java/com/google/security/zynamics/binnavi/data/postgresql_tables.sql instead of wrong one.

And I just fixed missing ALTER OWNER statements.

**Comment by EKarpulevich** [comment]
Currently sent for improvement.

**Pull request closed without merging**